client/gutenberg/extensions/map-block/add-point/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+import { Component } from '@wordpress/element';
+
+import {
+	Button,
+	Dashicon,
+	Popover
+} from '@wordpress/components';
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import LocationSearch from '../location-search';
+
+import './style.scss';
+
+export class AddPoint extends Component {
+
+	constructor() {
+		super( ...arguments );
+		this.onAddPoint = this.onAddPoint.bind( this );
```

Comment:
If we make `onAddPoint` an arrow function, it will be automatically binded and we won't need a `constructor` at all.

Anyway, `this.onAddPoint` seems to be unused, so we can directly remove the `constructor` too.

Reply:
BTW this is clearly a pattern coming from Gutenberg core blocks.

Reply:
Good idea, I prefer the arrow functions too - replaced all of them in https://github.com/Automattic/wp-calypso/pull/27836/commits/92aa832fd43ffac9ffdbc63ec43c68a9d9346760